src/Dapr.Actors/Runtime/ActorManager.cs

```diff
@@ -263,7 +263,11 @@ internal async Task ActivateActorAsync(ActorId actorId)
         private async Task<ActorActivatorState> CreateActorAsync(ActorId actorId)
         {
             this.logger.LogDebug("Creating Actor of type {ActorType} with ActorId {ActorId}", this.ActorTypeInfo.ImplementationType, actorId);
-            var host = new ActorHost(this.ActorTypeInfo, actorId, this.jsonSerializerOptions, this.loggerFactory, this.proxyFactory, this.daprInteractor);
+            var host = new ActorHost(this.ActorTypeInfo, actorId, this.jsonSerializerOptions, this.loggerFactory, this.proxyFactory)
+            {
+                DaprInteractor = this.daprInteractor,
+                StateProvider = new DaprStateProvider(this.daprInteractor, this.jsonSerializerOptions),
```

Comment:
The fix for #575 will be to do the same thing for reminders as we do for state. The Actor class is using the http interactor directly instead of a higher level abstraction, so it's not mockable.